Week1/arrays.js

```diff
@@ -0,0 +1,8 @@
+//append arrays
+let arraysInside = [];
+console.log ('my array will contain: eagle, horse and dog');
```

Comment:
Here the array does not contain those values **yet**, but it will contain them after line 5!

Reply:
updated